pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -1,45 +1,41 @@
 ---
 description: >-
-  This guide is for manually onboarding a parachain to the Polkadot "Paseo"
-  Testnet Locally
+  This guide shows how to manually onboard a chain to Paseo Local
 ---
 
-# Launching your parachain manually
+# Onboard a Chain to Paseo Local
 
-## Introduction
+[Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet. Paseo Local is for running Paseo on a local machine.
 
-[Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet.
+A typical development workflow for launching a chain on Polkadot:&#x20;
```

Comment:
Afraid of adding content to this already long doc. 
But I feel we could use some extra pointers for the bullet points bellow.

#1. Is very useful when testing the runtime of your chain. You are not really worried about how the onboarding process works. But you need to see the runtime executing some blocks. "live" integration tests with other chains and such.

#2. Is only helpful when you are trying to learn how to onboard your chain to an already running relay chain (like Westend, Paseo, Kusama or Polkadot). It is very helpful to nail down the concepts around bootstrapping collators, seeing the process and taking a few leanings from it.
This document.

The learnings in #2. are mostly equivalent to #3. in terms of what the user needs to know.
I'd argue that we can port most of the new content from here into 3. (or vice versa) and get rid of all the details around configuring  the relay chain. It is too nuanced, and possibly it changes in future versions of the SDK.

Ideally no user would need to know about the scheduling gymnastics the relay does.

Reply:
This was a bit hard to follow what you meant. Please see the latest changes and shout if there is anything pressing missing (or create an issue if not pressing).